atom/browser/resources/win/atom.manifest

```diff
@@ -32,7 +32,6 @@
 
   <asmv3:application xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
     <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">
-      <dpiAware>true</dpiAware>
```

Comment:
I can probably work on this and I also have a combination of High/Low DPI screens running.
I think `EnableNonClientDpiScaling` is really important (or `permonitorv2`), i wouldn't go for `true/pm` only